src/content/documentation/publish/source-code-submission.md

```diff
@@ -63,15 +63,22 @@ To reproduce the build, the reviewer runs the instructions you provided and then
 - details, including required version and installation instructions, of any tools or utilities that need to be downloaded, for example, [yuicompressor](http://yui.github.io/yuicompressor/).
 - a list of all the commands to generate an identical copy of the extension from the source code, for example, npm install or a grunt target. Ideally, you should include every command in the build script file.
 
+Reviewers will try to replicate your build environment if specified. However, if you do not specify the build environment, here's the default build environment that reviewers use:
+
+- Latest Ubuntu LTS (Desktop edition) virtualized via VirtualBox running on an x86_64 host
+- 10GB of system memory (RAM), 6 cores of vCPU
+- Latest Node LTS and corresponding npm
+- 50GB of disk space
```

Comment:
Is that free disk space after the OS has been installed?

Reply:
Good one, thanks!